etsi_its_conversion/etsi_its_conversion/src/Converter.cpp

```diff
@@ -462,22 +473,33 @@ void Converter::udpCallback(const UdpPacket::UniquePtr udp_msg) {
     else detected_etsi_type = "unknown";
   }
 
-  int msg_size = udp_msg->data.size() - etsi_message_payload_offset_;
-  ROS12_LOG(INFO, "Received ETSI message of type '%s' as bitstring (message size: %d | total payload size: %ld)", detected_etsi_type.c_str(), msg_size, udp_msg->data.size());
+  const uint8_t* protocol_version = reinterpret_cast<const uint8_t*>(&udp_msg->data[etsi_message_payload_offset_]);
 
-  if (detected_etsi_type == "cam") {
+  int msg_size = udp_msg->data.size() - etsi_message_payload_offset_;
+  ROS12_LOG(INFO, "Received ETSI message of type '%s' (protocolVersion: %d) as bitstring (message size: %d | total payload size: %ld)", detected_etsi_type.c_str(), *protocol_version , msg_size, udp_msg->data.size());
```

Comment:
We should check if the `protocol_version` of the received message matches the ones implemented in this repository (or at least write an Issue, that this feature is currently missing and could lead to unexpected behavior)

Reply:
Same applies for other messages